README.md

```diff
@@ -58,6 +58,11 @@ Additionally, the Hive plugin must be configured with location of your Hive meta
 
     -Dhive.metastore.uri=thrift://localhost:9083
 
+#### Common problems
+In newer IntelliJ versions the "VM options" field in the Run/Debug configuration may not be present by default. You can add it by clicking  "Modify options" and selecting "Add VM options". It is common to accidentally copy the VM options into the "Program arguments" field. Doing so results in errors on startup that start with "Explicit bindings are required ...". 
+
+If using Java 11 (or later) the compilation may fail with `java: package sun.misc does not exist`. Go to "Settings" → "Build, Execution, Deployment" → "Compiler" → "Java Compiler" and uncheck "use '--release' option for cross-compilation".
```

Comment:
```suggestion
* If using Java 11 (or later) the compilation may fail with `java: package sun.misc does not exist`. Go to "Settings" → "Build, Execution, Deployment" → "Compiler" → "Java Compiler" and uncheck "use '--release' option for cross-compilation".
```
Suggest making this third item part of the unordered list. The point highlights for the user where the beginning of a problem description is.